_episodes/07-github.md

```diff
@@ -141,6 +141,13 @@ To https://github.com/vlad/planets.git
 ~~~
 {: .output}
 
+In the above command we used a new word -- `master`. If the command completed successfully, the
+message that Git displays hints us that `master` is a _branch_. To put it simply, a branch is a
+sequence of changes ("commits") made to the repository. When we initialize a new repository, Git
+creates a new branch and, by default, calls it `master`. When we send ("push") our changes to the
+remote repository, we send "branches", not commits. So, in the above command we asked Git to send
+(`push`) changes in our only branch (`master`) to the remote called `origin`.
```

Comment:
This is a really wonderful "human terms" explanation!

Reply:
@maxim-belkin @munkm 
I'm concerned this might confuse new learners when looking at nomenclature [commonly used](https://help.github.com/en/github/using-git/using-common-git-commands) 
> When we send ("push") our changes to the remote repository, we send "branches", not commits.

what do you think about changing it to 
> When we send ("push") our changes to the remote repository, we push commits made on a local branch to the remote repository.

?

Reply:
`We push commits made on a local branch` is a valid way to say it, but it does not emphasize the fact that we send branches, not individual commits on a branch. Would saying "individual commits" rather than "commits" help?

> When we send ("push") our changes to the remote repository, we send "branches", not individual commits.

Or this:

> To send ("push") our changes to the remote repository, we have to specify the branch which we'd like to update on the remote repository.

Or maybe there is a better suggestion...

Reply:
Ok, maybe I'm adding to our choices here and overcomplicating things, but how would you both feel about 

> When we send ("push") our changes to the remote repository, we are sending the sequence of commits (also known as our "branch") rather than individual commits. 

? 

This echoes the earlier language in this PR about what a branch is while also (I think) taking language that you've both used in this discussion.

Reply:
> Ok, maybe I'm adding to our choices here and overcomplicating things, 

I think this is a good discussion.

> how would you both feel about
> 
> > When we send ("push") our changes to the remote repository, we are sending the sequence of commits (also known as our "branch") rather than individual commits.

We introduced branches a couple of seconds ago, so I don't think we need to repeat ourselves with `also known as our "branch"`. So, my suggestion would be:

> When we send ("push") our changes to the remote repository, we send sequences of commits ("branches") rather than individual commits.

Though, I do think it would be best to give learners an example of how to use the word "branch" in a sentence:

> When we send ("push") our changes to the remote repository, we send "branches", that is sequences of commits, rather than individual commits.

Reply:
How about 

> When we send ("push") our changes to the remote repository, we push the whole branch which includes all commits made on the local branch. Sometimes people will describe it as "pushing commits to the remote," but thinking of it in terms of sending the whole branch will help reduce unintended conflicts or changes.

Reply:
And, is this content in a callout box?  I think it might need to be.